modules/simple-payments/paypal-express-checkout.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * This PaypalExpressCheckout global is included by wp_enqueue_script( 'paypal-express-checkout' );
+ * It handles communication with Paypal Express checkout and public-api.wordpress.com for the purposes
+ * of simple-payments module.
+ */
+
+/* global paypal */
+/* exported PaypalExpressCheckout */
+/* jshint unused:false */
+var PaypalExpressCheckout = {
+	constants: {
+		createPaymentEndpoint: '', //TODO: point to the actual endpoint
+		executePaymentEndpoint: '' //TODO: point to the actual endpoint
+	},
+	renderButton: function( id ) {
+		paypal.Button.render( {
+			commit: true,
+			style: {
+				label: 'pay',
+				color: 'blue'
+			},
+			payment: function() {
+				return paypal.request.post( PaypalExpressCheckout.constants.createPaymentEndpoint ).then( function( data ) {
```

Comment:
Definitely out of the scope of this PR, lets add other stuff in another one.
This is monster already